test/integration/seeds/consent.test.ts

```diff
@@ -49,31 +49,37 @@ describe('testing Consent table', (): void => {
       id: '123',
       initiatorId: 'PISPA',
       participantId: 'DFSPA',
+      status: 'ACTIVE',
```

Comment:
@akapur99 shouldn't we reseed also some data with REVOKED status to have data ready for tests of `revoke` related logic?